### DISCUSSION TITLE
GraphQL API : how to filter by date (before)

### DISCUSSION THREAD

**Discussion by neo7-studio-web** [directus/directus#5072] [Archive]
Hi here is my request :

```
      query {
        article(
          filter: {
            status: { _eq: "published" },
            # published: { "_lte": "2021-04-13" }, // Doesn't seem work for the moment in Directus
          },
          sort:"published"
          ){
          title
          cover {
            id
          }
          cover_alt
          published
          content
          excerpt
          category
          serie
          tags {
            item {
              ... on tag {
                  id
                  label
              }
            }
          }
        }
      }
```
I would like to retrieve articles that are published before NOW. Is it possible ?

**Comment by RageBrahimi** [comment]
```javascript
query {
  articles(filter: { date_created: { _lt: "2021-04-15T14:28:00+01:00" } }) {
    id
    ...
  }
}
```
This will work on the basis that your collection is called `articles` and you have a `DateTime` item called `date_created` which is used as a reference for when it was created. More documentation [here](https://docs.directus.io/reference/api/items/#items-data-access).

**Reply by neo7-studio-web** [reply]
Thank you very much for your help. It doesn't work, though : 
![erreur](https://user-images.githubusercontent.com/18073828/114885138-ab8bf280-9e06-11eb-80c1-69d53c574ee9.jpg)

Note that the field "published" is managed and returned as a date not datetime

**Reply by rijkvanzanten** [reply]
@neo7-studio-web that's https://github.com/directus/directus/issues/4991 which was fixed and included in the latest release

**Reply by neo7-studio-web** [reply]
Working ! Thank you
